### DISCUSSION TITLE
Pricing for Github Actions to run ontime?

### DISCUSSION THREAD

**Discussion by danjefferies** [community/community#119823] [Actions] [answered]
### Select Topic Area

Question

### Body

I'm trying to automate a python script using cron and Github Actions to run at specific hours. I know the timing for Actions is sometimes delayed/off due to it being a queue system. Is there a premium plan available to get Github Actions to run at the exact hours I need it to? 

Open to other suggestions but I find AWS very confusing.

**Comment by grandmas-favorite** [comment, marked as answer]
Hi @danjefferies!

Per our [Actions billing info](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions), the pricing levels are benchmarked by processing minutes and not schedule - so there is no tier that allows for scheduling as you described. That said, it's an interesting idea, so I'll mark it as product feedback for ya!
